packages/contentful--app-scripts/utils/analytics.js

```diff
@@ -0,0 +1,33 @@
+const Analytics = require('analytics-node');
```

Comment:
May be use `import`
```suggestion
import Analytics from 'analytics-node';
```

Reply:
This project doesn't have a build process, therefore we can not use modules and need to use require